src/components/App/Post/PostQuickAdd.js

```diff
@@ -0,0 +1,62 @@
+import React, { Component } from 'react';
+import { uniqueId } from 'lodash';
+
+import { Input } from '../../Form';
+
+const generatePostUrl = () => {
+  const imgNumber = Math.floor(Math.random() * 100);
+  return `https://picsum.photos/320/240?image=${imgNumber}`;
+};
+
+class PostQuickAdd extends Component {
+  emptyPost = {
```

Comment:
I'd create a function called `_resetPost` and just call it in `constructor`/`componentDidMount` and bottom of `handleSubmit` function.

Or extract this into a constant outside the component. I think there's no need to assign instance variable for every component instance

Reply:
Yes, we talked about those approaches. Ill change to extracting constant outside the component to create less confusion.

I also got a question about using underscore prefixed functions like `_resetPost`, which IMHO goes much deeper. When would you make those functions vs regular vs external functions?